server/controllers/loginUser.js

```diff
@@ -0,0 +1,39 @@
+const jwt = require('jsonwebtoken');
+const loginQuery = require('../database/queries/loginQuery');
+const { comparePassword } = require('../utils/hashPassword');
+const getHashPassword = require('../database/queries/getHashPassword');
+require('env2')('.env');
```

Comment:
No need to require env file here as you're importing it at app.js, just omit this line.